distribution/sdk/sample-nsili-server/src/main/java/org/codice/alliance/nsili/mockserver/server/MockNsili.java

```diff
@@ -70,8 +86,49 @@ public void startWebServer(int port) {
         sf.create();
     }
 
+    public void startFtpWebServer(int port) {
+        FtpServerFactory ftpServerFactory = new FtpServerFactory();
+
+        ListenerFactory listenerFactory = new ListenerFactory();
+        listenerFactory.setPort(port);
+        ftpServerFactory.addListener("default", listenerFactory.createListener());
+
+        PropertiesUserManagerFactory propertiesUserManagerFactory =
+                new PropertiesUserManagerFactory();
+        UserManager userManager = propertiesUserManagerFactory.createUserManager();
+        BaseUser baseUser = new BaseUser();
+        baseUser.setName("admin");
+        baseUser.setPassword("admin");
+
+        try {
+            ftpHomeDirectoryPath = Files.createTempDirectory("home_");
+            Runtime.getRuntime()
+                    .addShutdownHook(new Thread(() -> FileUtils.deleteQuietly(ftpHomeDirectoryPath.toFile())));
+            baseUser.setHomeDirectory(ftpHomeDirectoryPath.toString());
+        } catch (IOException e) {
+            System.out.println("ERROR: Unable to set ftp endpoint to a temporary home directory.");
+        }
+
+        try {
+            userManager.save(baseUser);
+            ftpServerFactory.setUserManager(userManager);
+
+            ConnectionConfigFactory connectionConfigFactory = new ConnectionConfigFactory();
+            connectionConfigFactory.setAnonymousLoginEnabled(true);
```

Comment:
It doesn't look like this ConnectionConfigFactory is being used. Anonymous login looks to be enabled by default when a new DefaultFtpServerContext is made after calling `ftpServerFactory.createServer()`, so you should be able to get rid of these lines.